src/components/JobsPanelDataInputs/jobsPanelDataInputs.util.js

```diff
@@ -479,3 +457,20 @@ export const handleStoreInputPathChange = (
     payload: Boolean(projectItemReferenceIsEntered)
   })
 }
+
+export const isPathInputValid = (pathInputType, pathInputValue) => {
+  switch (pathInputType) {
+    case MLRUN_STORAGE_INPUT_PATH_SCHEME:
+      return /^(artifacts|feature-vectors)\/(.+?)\/(.+?)(#(.+?))?(:(.+?))?(@(.+))?$/.test(
+        pathInputValue
+      )
+    case HTTP_STORAGE_INPUT_PATH_SCHEME:
+    case HTTPS_STORAGE_INPUT_PATH_SCHEME:
+      return (
+        pathInputValue.split('/')[1] &&
+        pathInputValue.split('/')[1]?.length !== 0
+      )
```

Comment:
Don't forget you can use `?.[]`:
```suggestion
      return pathInputValue.split('/')?.[1]?.length > 0
```